README.md

```diff
@@ -235,6 +235,12 @@ Places with fish and boats.
 
 Marine areas are considered `common_optional`.
 
+### marketarea
+
+`1360666019`
+
+Market areas are considered `optional`.
```

Comment:
Please add a description sentence here as well:

> Economic geography areas that are usually an aggregation of counties or zipcodes based on TV, radio, and other media related audiences.

Reply:
Fixed in https://github.com/whosonfirst/whosonfirst-placetypes/pull/18/commits/4c91f8b271429e1e6ae36dee96600d201199625a